mirar/processors/photometry/aperture_photometry.py

```diff
@@ -131,9 +131,15 @@ def _apply_to_images(
                 suffix = self.col_suffix_list[ind]
                 image[f"fluxap{suffix}"] = flux
                 image[f"fluxunc{suffix}"] = fluxunc
-                image[f"magap{suffix}"] = float(
-                    image[self.zp_colname]
-                ) - 2.5 * np.log10(flux)
-                image[f"magerrap{suffix}"] = 1.086 * fluxunc / flux
+
+                magnitudes, magnitudes_unc = get_mags_from_fluxes(
+                    flux_list=[flux],
+                    fluxunc_list=[fluxunc],
+                    zeropoint_list=[float(image[self.zp_key])],
+                    zeropoint_unc_list=[float(image[self.zp_std_key])],
+                )
+
+                image[f"magap{suffix}"] = magnitudes[0]
```

Comment:
what are these? (magnitudes). If it's a dataframe you should use iloc. Or are they lists?

Reply:
Its a list.